README.md

```diff
@@ -293,6 +294,42 @@ data () {
 
 [Here](https://github.com/iliyaZelenko/tiptap-vuetify/issues/100#issuecomment-551950075) is example of how to create your extension from scratch.
 
+### custom image upload components
+A custom image upload / selection component allows you to upload images to or select images from your application's backend system.
+The when properly configured, the component will be displayed as a tab in the Add Image window.
+
+To implement this, first create a component where users can upload and/or select images. The component will not get any props from the image window.
+When a user selects an image, the component must emit a `select-file` event with an object containing `src` and `alt` properties.
+For example:
+```js
+selectImage() {
+  this.$emit('select-file', { src: '/path/to/image.jpg', alt: 'Uploaded image' });
```

Comment:
```suggestion
  // There may be your asynchronous file upload to the server (backend), when the file is uploaded you can set the path to the "src" property.
  this.$emit('select-file', { src: '/path/to/image.jpg', alt: 'Uploaded image' });
```

Reply:
I get what your saying, better wording might be
// When doing an asynchronous  upload, you can set the src property to the value provided by the server (backend).

Reply:
You are right. Done [here](https://github.com/iliyaZelenko/tiptap-vuetify/commit/1e302f85b04f9d855388541aa2723feeb2a9b645).